packages/fluentui/docs/src/examples/components/SvgIcon/Variations/SvgIconExampleBordered.shorthand.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { Flex } from '@fluentui/react-northstar';
+import {
+  SkypeLogoIcon,
+  EmojiIcon,
+  PlayIcon,
+  CallIcon,
+  CallVideoIcon,
+  TranslationIcon,
+} from '@fluentui/react-icons-northstar';
+
+const IconExampleBordered = () => (
+  <Flex gap="gap.smaller">
+    <SkypeLogoIcon bordered />
+    <EmojiIcon bordered />
+    <PlayIcon bordered />
+    <CallIcon bordered />
+    <CallVideoIcon bordered />
+    <TranslationIcon bordered />
+  </Flex>
+);
+
+export default IconExampleBordered;
```

Comment:
Should be all exports in examples renamed to `SvgIcon*`?

Reply:
BTW these examples are not going to be added on the docs for now, will be with next PR (due to folder changes required for the Icon component..)

Reply:
Done